packages/authenticator/amplify_authenticator/lib/src/mixins/authenticator_phone_field.dart

```diff
@@ -13,51 +13,51 @@ import 'package:flutter/material.dart';
 mixin AuthenticatorPhoneFieldMixin<FieldType extends Enum,
         T extends AuthenticatorFormField<FieldType, String>>
     on AuthenticatorFormFieldState<FieldType, String, T>
-    implements SelectableConfig<CountryResolverKey, Country> {
-  late final CountryResolver _countriesResolver = stringResolver.countries;
+    implements SelectableConfig<DialCodeResolverKey, DialCode> {
+  late final DialCodeResolver _dialCodeResolver = stringResolver.dialCodes;
 
   @override
-  Country get selectionValue => state.country;
+  DialCode get selectionValue => state.dialCode;
 
   String _searchVal = '';
 
   @override
-  late final List<InputSelection<CountryResolverKey, Country>> selections =
-      countryCodes
+  late final List<InputSelection<DialCodeResolverKey, DialCode>> selections =
+      DialCode.values
           .map(
-            (Country country) => InputSelection(
+            (DialCode country) => InputSelection(
               label: country.key,
               value: country,
             ),
           )
           .toList();
 
-  List<Country> get filteredCountries => countryCodes
+  List<DialCode> get filteredCountries => DialCode.values
       .where(
-        (country) => _countriesResolver
+        (country) => _dialCodeResolver
             .resolve(context, country.key)
             .toLowerCase()
             .contains(_searchVal.toLowerCase()),
       )
-      .sortedBy(
-        (country) => _countriesResolver.resolve(context, country.key),
-      )
-      .toList();
+      .toList()
+    ..sortedBy(
```

Comment:
Unlike `sort` which returns `void` and acts in-place on the list, `sortedBy` returns the sorted list and, thus, cannot be cascaded.

Reply:
And since `sortedBy` works on any `Iterable`, you do not need to call `iterable.toList().sortedBy(...)` which would allocate two lists. Instead, you can just call `iterable.sortedBy(...)` and this will return the sorted list.